src/components/DaysPanel/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { IonContent, IonItem, IonLabel, IonList, IonRadioGroup, IonRadio } from '@ionic/react';
+import React, { useState } from 'react';
+import  ActionableModalCard  from '../ModalCards/ActionableModalCard'
+import './style.scss';
+import store from '../../store';
+
+
+
+
+
+const DaysPanel : React.FC = () => {
+    
+    const[onHoldProps,setOnHoldProps] = useState( {showModal: false, days : [''], selectedDay:'' } );
+  
+    let timer = 0;
+    let days;
+    
+
+    let handleOnMouseDown = ( day: string ) => 
+      timer = new Date().getTime();   
+    
+
+    let handleOnMouseUp = (day: string) => {
+        let totalTime = (new Date().getTime() - timer)/1000;  
+        if(totalTime > 2)     
+            setOnHoldProps({showModal: true ,days: onHoldProps.days, selectedDay: day});    
+   }
+
+    
+    let markNonWorkingDay = (day: string)=> {
+        store.getData().then(data => {
+            data[day]['isWorkDay'] = false;

```

Comment:
['isWorkDay'], square brackets should be avoided as it skips typescript from checking types, making typescript useless